plugins/metrics/pkg/cortex/api.go

```diff
@@ -25,6 +25,7 @@ type forwarders struct {
 	QueryFrontend gin.HandlerFunc
 	Alertmanager  gin.HandlerFunc
 	Ruler         gin.HandlerFunc
+	RemoteRead    gin.HandlerFunc
```

Comment:
Unused?

Reply:
yep not used, I'm gonna do another once over to catch anything that I forget about when I was doing the initial integration and didn't know where things went